src/AppBundle/Entity/Report/Report.php

```diff
@@ -441,8 +441,8 @@ public function __construct(Client $client, $type, \DateTime $startDate, \DateTi
         }
         $this->type = $type;
         $this->client = $client;
-        $this->startDate = new \DateTime($startDate->format('Y-m-d'));
-        $this->endDate = new \DateTime($endDate->format('Y-m-d'));
+        $this->startDate = new \DateTime($startDate->format('Y-m-d'), new \DateTimeZone('Europe/London'));
+        $this->endDate = new \DateTime($endDate->format('Y-m-d'), new \DateTimeZone('Europe/London'));
```

Comment:
?

Reply:
I was getting test failure where it complained about using default timezones so I set them here and in the test.